### PR TITLE
[CP] Admin & form UI

### PR DESCRIPTION
**Pull request by Remg**
CP admin:
- add `featured` tag
- add `matchedSkills` tag
- add `adminComment` editable text
- change columns & filters on list view

CP front:
 - add counter on `assistance_content` field
 - change `phone` field place

### CODE REVIEW

File: src/Repository/CitizenActionRepository.php [hunk 1]
```diff
@@ -10,6 +12,18 @@ class CitizenActionRepository extends EventRepository
     const TYPE_UPCOMING = 'upcoming';
     const TYPE_ALL = 'all';
 
+    public function findNextCitizenActionForCitizenProject(CitizenProject $citizenProject): ?CitizenAction
+    {
+        return $this
+            ->createQueryBuilder('a')
```
Comment (thomasbisignani) [inline src/Repository/CitizenActionRepository.php]: Can you use a bigger/explicit alias ?

File: templates/admin/citizen_project/list_name.html.twig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
+
+{% block field %}
+    <a href="{{ path('admin_app_citizenproject_edit', {'id': object.id}) }}">
```
Comment (thomasbisignani) [inline templates/admin/citizen_project/list_name.html.twig]: Simples quotes are useless for keys

File: templates/admin/citizen_project/list_next_citizen_action.html.twig [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
+
+{% block field %}
+    {% if object.nextAction %}
+        <a href="{{ path('admin_app_citizenaction_show', {'id': object.nextAction.id}) }}">
```
Comment (thomasbisignani) [inline templates/admin/citizen_project/list_next_citizen_action.html.twig]: Same here

File: templates/admin/citizen_project/members.html.twig [hunk 4]
```diff
@@ -95,10 +88,12 @@
                                     <span class="label label-default">Membre</span>
                                 </td>
                                 <td>
-                                    <a href="{{ path('app_admin_citizenproject_change_privilege', {citizen_project: citizen_project.id, adherent: member.id, privilege: constant('CITIZEN_PROJECT_ADMINISTRATOR', membership), token: token}) }}"
-                                       class="btn btn-default btn-sm">
-                                        Transformer en administrateur
-                                    </a>
+                                    {% if citizen_project.status is same as(constant('APPROVED', citizen_project)) %}
```
Comment (thomasbisignani) [inline templates/admin/citizen_project/members.html.twig]: I think you can use this function here: ` {% if citizen_project.isApproved %}`

File: templates/citizen_project/_form.html.twig [hunk 5]
```diff
@@ -155,3 +144,15 @@
         }
     }) }}
 </div>
+
+<div class="l__row l__row--top l__row--h-stretch l__mobile--col" id="citizen-project-phone">
+    <div class="l__col l__col--top register__form__phone">
+        {{ form_label(form.phone, 'Téléphone', { 'label_attr': {'class': 'text--gray'} }) }}
```
Comment (thomasbisignani) [inline templates/citizen_project/_form.html.twig]: Same here for quotes and keys

File: templates/admin/citizen_project/members.html.twig (hunk 4 above)
Comment (Remg) [reply templates/admin/citizen_project/members.html.twig]: :ok_hand:

File: templates/citizen_project/_form.html.twig [hunk 6]
```diff
@@ -155,3 +144,15 @@
...
+        <div class="l__row l__row--top l__row--h-stretch">
+            {{ form_widget(form.phone.country) }}
+            {{ form_widget(form.phone.number, { attr: { placeholder: 'Téléphone' } }) }}
```
Comment (thomasbisignani) [inline templates/citizen_project/_form.html.twig]: Whites spaces are useless in arrays

**Comment by thomasbisignani** [comment]
:ok_hand:

**Pull request merged**